controllers/apply/awards-for-all/fields.js

```diff
@@ -1735,8 +1864,17 @@ module.exports = function fieldsFor({ locale, data = {} }) {
             isRequired: false,
             schema: Joi.string()
                 .allow('')
+                .max(FREE_TEXT_MAXLENGTH.large)
                 .optional(),
-            messages: []
+            messages: [
+                {
+                    type: 'string.max',
+                    message: localise({
+                        en: `The Religion or belief must be ${FREE_TEXT_MAXLENGTH.large} characters or less`,
```

Comment:
Is this for the 'Other' field on the **Religion or belief (Step 7 of 9)** page? If so:

`Other religions or beliefs must be ${FREE_TEXT_MAXLENGTH.large} characters or less`